integrations/cross_platform/android.rst

```diff
@@ -0,0 +1,196 @@
+.. _android:
+
+|android_logo| Android
+____________________________
+
+There are several ways to cross-compile packages for `Android <https://www.android.com/>`__ platform via conan.
+
+Using android_ndk_installer
+===========================
+
+The easiest way so far is to use `android_ndk_installer <https://github.com/bincrafters/conan-android_ndk_installer>`_ conan package.
+
+Referencing mentioned package as build requirement will do the following steps:
+
+- Download the appropriate `Android NDK <https://developer.android.com/ndk>`_ archive.
+
+- Set up required environment variables, such as ``CC``, ``CXX``, ``RANLIB`` and so on to the appropriate tools from the NDK.
+
+- For the CMake, will use appropriate `toolchain file <https://developer.android.com/ndk/guides/cmake#the_cmake_toolchain_file>`_ and set up necessary CMake `variables <https://developer.android.com/ndk/guides/cmake#variables>`_.
```

Comment:
```suggestion
- In case of using CMake, it will inject the appropriate `toolchain file <https://developer.android.com/ndk/guides/cmake#the_cmake_toolchain_file>`_
and set up the necessary CMake `variables <https://developer.android.com/ndk/guides/cmake#variables>`_.
```